embrace_mobile/manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+  "display_name": "Embrace Mobile",
+  "maintainer": "support@embrace.io",
+  "manifest_version": "1.0.0",
+  "name": "embrace_mobile",
+  "metric_prefix": "embrace_mobile.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Mobile observability for iOS, Android, React Native, and Unity",
+  "guid": "745d1d31-cce9-49b3-ae99-937209967741",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Embrace Mobile",
+  "categories": [
+    "monitoring", "network", "exceptions", "marketplace", "metrics", "alerting", "issue tracking", "log collection"
```

Comment:
Is this integration going to include logs? One of the CI validations is failing because `log collection` is included as a category despite no logs configurations in the check.

Reply:
No, I can go ahead and remove that.